.gitignore

```diff
@@ -21,3 +21,7 @@
 package/*.tar.bz2
 
 vendor/
+coverage/
+public/repo/
+
+.idea
```

Comment:
that is developer specific - should generally live in your personal gitignore, not the project one.